### DISCUSSION TITLE
Setting past month into read-only mode

### DISCUSSION THREAD

**Discussion by StephenWilde** [kimai/kimai#3374] [General]
Hello together,

is there any function in Kimai that can be used to set booked month in the past into a read-only state?

I mean something like: 

A deadline for the last month until someone can still do changes in this month. But after this deadline, the month will be in read-only mode, so that users can´t change or add further actions and time?

Or it would be enough, if I as an Admin would have a function to lock a month and set it to read-only with it.

Thx
Stephen

**Comment by kevinpapst** [comment]
Check lockdown system settings, there are docs existing.

**Comment by StephenWilde** [comment]
Thx very much! It works
